cde-core/resource/resources/custom/components/NewMapComponent/NewMapComponent.js

```diff
@@ -522,9 +522,9 @@ var NewMapComponent = (function () {
       // Define default shape appearance
       var shapeSettings = _.defaults(this.shapeSettings || {}, {
         //fillColor:  'blue',
-        //fillOpacity: 0.5,
+        fillOpacity: this.shapeFillOpacity,
         strokeWidth: 2,
-        strokeColor: 'white',
+        strokeColor: this.shapeStrokeColor,
         zIndex: 0
       });
 
```

Comment:
If the user fails to define the shapeFillOpacity and shapeStrokeColor, there is no fallback. It would be best to have:

var shapeSettings = _.defaults(this.shapeSettings || {}, {
    fillOpacity: this.shapeFillOpacity,
    strokeColor: this.shapeStrokeColor
},{
         //fillColor:  'blue',
        //fillOpacity: 0.5,
         strokeWidth: 2,
        strokeColor: 'white',
         zIndex: 0
       });